src/plugins/intel_cpu/tests/unit/ngraph_transformations/logsoftmax_decompose.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <memory>
+
+#include <openvino/opsets/opset1.hpp>
+#include <openvino/opsets/opset8.hpp>
+#include <ngraph_transformations/op/fully_connected.hpp>
+#include <ngraph_transformations/convert_logsoftmax.hpp>
+#include <ngraph_transformations/fc_bias_fusion.hpp>
+#include <transformations/init_node_info.hpp>
+#include <transformations/utils/utils.hpp>
+#include <openvino/pass/manager.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+using namespace testing;
+using namespace ov::intel_cpu;
+
+TEST(TransformationTests, ConvertLogSoftmax1) {
+    std::shared_ptr<ov::Model> f(nullptr), f_ref(nullptr);
```

Comment:
Please use TransformationTestsF that already contains function, function_ref, and manager fields
```suggestion
TEST(TransformationTestsF, ConvertLogSoftmax1) {
```
As example you could use [this file](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/unit/ngraph_transformations/move_eltwise_up_data_movement_test.cpp)